DigitalLearningSolutions.Web/Views/Signposting/LinkLearningHubSso/Index.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.ViewModels.Signposting.LinkLearningHubSso
+@model LinkLearningHubViewModel
+
+@{
+    ViewData[LayoutViewDataKeys.Title] = "Recommended Learning Resources - Account linked";
+    ViewData[LayoutViewDataKeys.Application] = "Recommended Learning Resources";
+    ViewData[LayoutViewDataKeys.HeaderPrefix] = "";
+    ViewData[LayoutViewDataKeys.TransactionalHeader] = true;
+    ViewData[LayoutViewDataKeys.DoNotDisplayNavBar] = true;
+    var actionLinkRouteData = new Dictionary<string, string> { { "resourceId", Model.ResourceLinkId?.ToString() } };
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+
+    <h1 class="nhsuk-heading-xl">Account linked</h1>
+
+    @if (Model.ShowIsAlreadyLinkedWarning)
+    {
+      <p>Your account has already been linked with the Learning Hub in a previous link request.</p>
+      <p>Therefore no changes have been made to your account regarding this link request.</p>

```

Comment:
I think we could make this a bit more concise (Eg. `Your account is already linked with Learning Hub. \n No change has been made.`), but let's check with HEE when we demo it.

Reply:
I've changed it as per the suggestion but we can still bring this up.